app/helpers/helpers/helpers_file.go

```diff
@@ -8,6 +8,39 @@ import (
 	"strings"
 )
 
+func GetCommand(line string) string {
```

Comment:
the parameter is called here `line` versus `test` in the below helpers
is this intended?
is it something different we are passing?

Reply:
addressed in d9c081de52e70f86c4ee0af1d15f3af69343fabf